CMakeLists.txt

```diff
@@ -23,7 +23,7 @@
 # SOFTWARE.
 #
 ################################################################################
-cmake_minimum_required( VERSION 3.11 )
+cmake_minimum_required(VERSION 2.8.12...3.11)
```

Comment:
The `2.8.12...3.11` syntax is only supported in cmake 3.12 or higher, so the versions listed dont make since. I think we can just change it to 3.10 or 3.16(I think thats the version in ubuntu 20.04).

Reply:
Ideally, it would be better to move the cmake versions changes to a seperate PR.

Reply:
Settle to 3.11 to be consistent with MIOpen :)